doc/collision_prediction_report.Rmd

```diff
@@ -0,0 +1,46 @@
+---
+title: "Predicting fatalities resulting from motor vehicle collisions"
+author: "MDS-2021-22 block3 group21"
+date: "25/11/2021"
+output: 
+  html_document:
+    toc: true
+  github_document:
+    toc: true
+bibliography: collision_prediction_references.bib
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = FALSE)
+library(knitr)
+library(kableExtra)
+library(tidyverse)
+```
+
+```{r load model results}
+final_model_scores <- read_csv("../results/scores_test2.csv", show_col_types = FALSE)
+```
+
+# Introduction
+
+Stemming from the daily news we come across on fatalities due to motor vehicle collisions, we thought about designing a model that would help predict whether a motor vehicle collision would result in a fatality or not. This lead to our predictive research question: "Will a motor vehicle collision result in fatalities?". But would it suffice to only "predict" a fatality? What if we also determine some of the major contributors that could lead to a fatal collision? Through this predictive analysis we first aim to predict whether a collision would result in a fatality or not followed by determining what features are most important in making these predictions. Specifically, we want to know how important month, roadway configuration, weather condition, person age, and vehicle type are in predicting the severity of a motor vehicle collision.
+
+# Analysis
+## Data
+The data set that will be used in this project has been sourced from the National Collision Database, published by Transport Canada, which can be found [here](https://open.canada.ca/data/en/dataset/1eb9eba7-71d1-4b30-9fb1-30cbdab7e63a). The National Collision Database contains data on all of the police-reported motor vehicle collisions on public roads in Canada from 1999 to the most recent available data from 2017. We will start off our analysis using the data collected from collisions that occurred in 2017. This data set contains information licensed under the Open Government Licence – Canada.
```

Comment:
...sourced from the National Collision Database (@NCDB)

Reply:
Updated reference and removed the direct link